### ISSUE TITLE
Clarification on the self.update function?

### ISSUE THREAD

**Issue by ytl41** [JustGlowing/minisom#155]
Hi, i have an issue where by i run the self.update function and it turns my som into a none type object, may i clarify the usage of the function?

Here is what i wrote:

X_train_som, X_test_som, y_train_som, y_test_som = train_test_split(data_som, som_class, test_size= 0.3, shuffle= False)
starting_som_weights = som.get_weights().copy()
w= som.winner(X_train_som[0])
som = som.update(starting_som_weights, w, 1, 100) 

Thanks!

**Comment by JustGlowing** [comment]
The update method doesn't return anything. Just call it without overwriting
anything.

**Comment by ytl41** [comment]
ah i see ok thanks!
